src/server/modules/user/sql.js

```diff
@@ -18,12 +18,15 @@ export default class User {
         'up.last_name',
         'ca.serial',
         'fa.fb_id',
-        'fa.display_name'
+        'fa.display_name',
+        'ga.google_id',
+        'ga.display_name'
```

Comment:
I think this will override the facebook `display_name` since both tables are returning the same column name.

Reply:
Yeah, I think you are right. Any thoughts, @verdverm  ?

Reply:
We need to give them aliases

Reply:
This will probably fix it https://github.com/sysgears/apollo-universal-starter-kit/commit/3bc8293e0d0470676b69ec8e57aa7dc00d7a7f47